packages/components/src/interfaces/native/Text.ts

```diff
@@ -28,16 +28,25 @@ const ItalicMarkSchema = Type.Object(
   },
 )
 
-const LinkMarkSchema = Type.Object(
+const BaseLinkMarkSchema = Type.Object(
   {
     type: Type.Literal("link", { default: "link" }),
-    href: Type.String(),
+    attrs: Type.Object({
+      target: Type.Literal("_blank"),
```

Comment:
issue: hmm this property should be optional

```suggestion
      target: Type.Optional(Type.Literal("_blank")),
```

Reply:
i don't think that's quite right because we're using tiptap's link schema. they default it to the `options.target`, which is `_blank` by default, see ss

<img width="442" alt="image" src="https://github.com/user-attachments/assets/0c29170a-6fec-4c67-995c-94004e051366">

Reply:
Hmm but we are not using Tiptap's link schema entirely. In HTML it is entirely possible to omit the `target` attribute and the link will have a default behaviour of `_self`. This schema means that all links must have `_blank`, which may not be the case for internal links.

Reply:
you're right! i misunderstood the schema here because i thought it was only validating tiptap. i'll set it to `optional("_self" | "_blank")` wdyt?